src/cargo-fmt/main.rs

```diff
@@ -245,17 +259,27 @@ pub struct Target {
     kind: String,
     /// Rust edition for this target.
     edition: String,
+    // Rust files residing within subdirectories of the tests directory.
```

Comment:
We can make this as doc comment:

```suggestion
    /// Rust files residing within subdirectories of the tests directory.
```